python/cudf/cudf/core/column/decimal.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+import cupy as cp
+import numpy as np
+import pyarrow as pa
+
+from cudf import _lib as libcudf
+from cudf.core.buffer import Buffer
+from cudf.core.column import ColumnBase
+from cudf.core.dtypes import DecimalDtype
+from cudf.utils.utils import pa_mask_buffer_to_mask
+
+
+class DecimalColumn(ColumnBase):
```

Comment:
If we're going to have explicit decimal32 / decimal64 columns and types I'm wondering if it makes sense to make that split now instead of merging with `DecimalColumn` and renaming to `Decimal64Column` later. Thoughts?

Reply:
I think a single `DecimalColumn`, but with distinct `Decimal32Dtype` and `Decimal64Dtype` should suffice (similar to what we have with NumericalColumn)